test/modules/op/avg_pool2d.py

```diff
@@ -41,9 +41,9 @@ def get_example_inputs(self):
         return (torch.randn(1, 128, 7, 7),)
 
     # This test should be done after applying NCHW_to_NHWC with onecc.
```

Comment:
Why does this test fail Without NCHW_to_NHWC ? It might be inefficient due to Transpose Ops, but shouldn't the value test pass?

Reply:
@jinevening I guess the axis values of Mean op are not supported in our backend compiler.